src/icrar/leap-accelerate/algorithm/cuda/PhaseRotate.cu

```diff
@@ -201,7 +202,7 @@ namespace cuda
         Eigen::MatrixXd dIntColumn = dInt(Eigen::all, 0); // 1st pol only
         assert(dIntColumn.cols() == 1);
 
-        cal.push_back(ConvertMatrix(Eigen::MatrixXd((hostMetadata.GetAd() * dIntColumn) + cal1)));
+        auto cal = Eigen::MatrixXd((hostMetadata.GetAd() * dIntColumn) + cal1);
         output_calibrations.emplace_back(direction, cal);
```

Comment:
I hadn't realized before, but you can probably `std::move(cal)`  then emplacing it into `output_calibrations`. Alternatively you can do `Eigen::MatrixXd((hostMetadata.GetAd() * dIntColumn) + cal1)` directly within the `output_calibrations.emplace_back`, or even simply `(hostMetadata.GetAd() * dIntColumn) + cal1)` like you do in `cpu::PhaseRotate`